src/System.Runtime/tests/System/ByteTests.cs

```diff
@@ -155,6 +155,8 @@ public static void ToString(byte b, string format, IFormatProvider provider, str
         public static void ToString_InvalidFormat_ThrowsFormatException()
         {
             byte b = 123;
+            Assert.Throws<FormatException>(() => b.ToString("R")); // Invalid format
+            Assert.Throws<FormatException>(() => b.ToString("R", null)); // Invalid format
```

Comment:
Should we test 'r' as well explicitly?

Reply:
Yes, testing both upper and lower would be good.

Reply:
Fixed.